osgpkitools/ConnectAPI.py

```diff
@@ -167,9 +169,9 @@ def retrieve_unauthenticated(self, **arguments):
         params = urllib.urlencode({'host_request_id': arguments['id']})
         headers = {'Content-type': arguments['content_type'],
                    'User-Agent': USER_AGENT}
-        conn = httplib.HTTPConnection(arguments['host'])
+        conn = httplib.HTTPSConnection(arguments['hostsec'])
```

Comment:
Update docstring above to specify `hostsec` instead of `host`. (Also, is it really a URL?)

Reply:
Yea, that's what [httplib](https://docs.python.org/2/library/httplib.html#httpconnection-objects) calls it